### DISCUSSION TITLE
Omnivore2trilium

### DISCUSSION THREAD

**Discussion by 0xbismarck** [omnivore-app/omnivore#3783] [Show and tell]
Hello:

I want to share with the community my new tool for exporting highlights from Omnivore, a Read-It-Later App, and sending the highlights to Trilium.
 - https://github.com/0xbismarck/omnivore2trilium

~0xB

**Comment by jacksonh** [comment]
hey I have posted this on our Mastodon, twitter, and Discord. Any chance you have a blog post about it we could repost?

**Comment by 0xbismarck** [comment]
hey there. 

No, I haven't made a formal blog post.
